src/main/java/org/opentripplanner/middleware/controllers/api/LogController.java

```diff
@@ -32,20 +37,45 @@ public static void register(Service spark, String apiPrefix) {
      */
     private static List<GetUsageResult> getUsageLogs(Request req, Response res) {
         // keyId param is optional (if not provided, all API keys will be included in response).
```

Comment:
Move this comment to under `if (apiKeys.isEmpty())`

Reply:
Updated.